google-cloud-irm/lib/google/cloud/irm/v1alpha2/helpers.rb

```diff
@@ -42,20 +42,20 @@ def incident_path project, incident
             self.class.incident_path project, incident
           end
           
-          # Alias for Google::Cloud::Irm::V1alpha2::IncidentServiceClient.project_path.
-          # @param project [String]
+          # Alias for Google::Cloud::Irm::V1alpha2::IncidentServiceClient.incident_role_assignment_path.
+          # @param project_id_or_number [String]
+          # @param incident_id [String]
+          # @param role_id [String]
           # @return [String]
-          def project_path project
-            self.class.project_path project
+          def incident_role_assignment_path project_id_or_number, incident_id, role_id
```

Comment:
@dazuma This renaming of `role_assignment_path` to `incident_role_assignment_path` appears to be a breaking change. Can you confirm that is change is desired by the product owner?

Reply:
We haven't actually released an IRM library yet (and we're being asked to hold off on releases for now). So I'm inclined just to accept any changes, including breaking ones, until we're told to prepare it for release.